dmriprep/utils/images.py

```diff
@@ -68,3 +135,232 @@ def median(in_file, dtype=None, out_path=None):
         dtype = hdr.get_data_dtype()
     nb.Nifti1Image(median_data.astype(dtype), img.affine, hdr).to_filename(out_path)
     return out_path
+
+
+def average_images(images, out_path=None):
```

Comment:
Arguably, for consistency, this should be called "average" (to match "median" above). 

Or better: let's make one function that summarizes images on the last dimension and takes a callable as input. If the callable is `np.median`, you get the median on the last dimension. If it's `np.mean`, then you get the mean. And if your feeling adventurous and pass your own callable here, it gives you back whatever that callable computes, provided it has the same API as `np.median` and `np.mean`, etc. 

@josephmje : what do you think about that?

Reply:
I guess we can move ahead with this for now and also revisit later.

Reply:
I agree with this. @josephmje What do you think?

Reply:
This sounds good to me. Especially since both functions are nearly identical.

Reply:
Now implemented in 7767498